tensor.go

```diff
@@ -5,46 +5,55 @@ package gotorch
 // #cgo LDFLAGS: -L ${SRCDIR}/cgotorch/libtorch/lib -Wl,-rpath ${SRCDIR}/cgotorch/libtorch/lib -lc10 -ltorch -ltorch_cpu
 // #include "cgotorch.h"
 import "C"
+import "runtime"
 
 // Tensor wrappers a pointer to C.Tensor
 type Tensor struct {
 	T C.Tensor
 }
 
 // String returns the Tensor as a string
-func (a Tensor) String() string {
+func (a *Tensor) String() string {
```

Comment:
Why do we need to change to use * here?

Reply:
Since we return `*Tensor` with `RandN` and  `*Optimizer` with `NetSGDOpt`, it's better to keep consistent.

version 1:

```go
a := torch.RandN(100, 10, false)
(*a).String()
```

version 2:
```go
a := torch.RandN(100, 10, false)
a.String()
```

Version 2 is simpler.